apps/demo/src/index.tsx

```diff
@@ -1,17 +1,14 @@
-import 'react-app-polyfill/stable';
-
 import { StrictMode } from 'react';
 import ReactDOM from 'react-dom';
 
-import 'normalize.css';
```

Comment:
Moved to `styles.css` for consistency with a change I had to make in the @h5web/app package to import React-Reflex' global styles.